data/en/isdateobject.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name":"isDateObject",
+	"type":"function",
+	"syntax":"isDateObject(value)",
+	"returns":"boolean",
+	"related":["isDate","isObject"],
+	"description":"Checks if the given value is a date / time object. The difference between isDateObject and isDate, latter returns true for date / time objects as well as date / time strings.",
```

Comment:
The grammar here seems off. Maybe something like "The difference between `isDateObject` and `isDate` is that the latter returns..."

Reply:
Have made the change and committed.